src/internal/connector/data_collections.go

```diff
@@ -213,15 +217,16 @@ func (gc *GraphConnector) ConsumeRestoreCollections(
 		return nil, clues.Wrap(err, "malformed azure credentials")
 	}
 
-	switch selector.Service {
+	switch sels.Service {
 	case selectors.ServiceExchange:
+		ctx = graph.BindRateLimiterConfig(ctx, graph.LimiterCfg{Service: path.ExchangeService})
```

Comment:
@ryanfkeepers rebase didn't go through it seems? This change is from an older PR.